From: Niko Tyni <ntyni@debian.org>
Subject: Disable zlib bundling in Compress::Raw::Zlib

Compress::Raw::Zlib statically links its bundled version of zlib
by default, but we use the system library instead.

---
 cpan/Compress-Raw-Zlib/config.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/cpan/Compress-Raw-Zlib/config.in b/cpan/Compress-Raw-Zlib/config.in
index c56cc03..2c6659b 100644
--- a/cpan/Compress-Raw-Zlib/config.in
+++ b/cpan/Compress-Raw-Zlib/config.in
@@ -16,9 +16,9 @@
 #    Setting the Gzip OS Code
 #
 
-BUILD_ZLIB      = True
-INCLUDE         = ./zlib-src
-LIB             = ./zlib-src
+BUILD_ZLIB      = False
+INCLUDE         = /usr/include
+LIB             = /usr/lib
 
 OLD_ZLIB        = False
 GZIP_OS_CODE    = AUTO_DETECT
-- 
tg: (c823880..) debian/disable-zlib-bundling (depends on: upstream)
